UKHO.ExternalNotificationService.API/UKHO.ExternalNotificationService.API/Controllers/WebhookController.cs

```diff
@@ -44,10 +48,38 @@ public virtual async Task<IActionResult> Post()
         {
             using var reader = new StreamReader(Request.Body, Encoding.UTF8);
             string jsonContent = await reader.ReadToEndAsync();
+            CustomEventGridEvent customEventGridEvent = JsonConvert.DeserializeObject<CustomEventGridEvent>(jsonContent);
```

Comment:
Yes , There is a challenge incoming event to ENS that isn't in the CloudEvent format.  
This data will not be able to bind the property in the CloudEvent class because it was found in an uppercase.
As discussed, I've created a custom class to make consistency.